canonical-web-and-design/ubuntu.com
# Integrate esm apps

## Done

- Apply IA changes to integrate ESM apps into the site

Here is a detailed list of the changes 
https://github.com/canonical-web-and-design/web-squad/issues/2346#issuecomment-721062383

~I only omitted the `/pricing and /pricing/infra` and the `legal` section because I wasn't exactly clear on what to do and was missing copy @matthewpaulthomas~


## QA

- go to https://ubuntu-com-8926.demos.haus/esm or https://ubuntu-com-8926.demos.haus/esm-apps
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the links are working correctly
- Check that all the mentions of ESM are changed to ESM Infra

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2347
